filter/filtering.go

```diff
@@ -0,0 +1,68 @@
+// Package filter provides filter model
+package filter
+
+import (
+	"database/sql"
+
+	"github.com/google/uuid"
+)
+
+type Postgres struct {

```

Comment:
It would be great to create interface with methods to get, update and delete filters
And let's rename package to `repository`. WDYT?

Reply:
You are right) I fixed it, but to tell the truth, still not sure I've named interface correct.